config/main.py

```diff
@@ -73,16 +70,17 @@ def _change_bgp_session_status_by_addr(ipaddress, status, verbose):
     config_db = ConfigDBConnector()
     config_db.connect()
 
-    # Convert the IP address to uppercase because IPv6 addresses are stored
-    # in ConfigDB with all uppercase alphabet characters
-    config_db.mod_entry('bgp_neighbor', ipaddress.upper(), {'admin_status': status})
+    config_db.mod_entry('bgp_neighbor', ipaddress, {'admin_status': status})
 
 def _change_bgp_session_status(ipaddr_or_hostname, status, verbose):
     """Start up or shut down BGP session by IP address or hostname
     """
     ip_addrs = []
-    if _is_neighbor_ipaddress(ipaddr_or_hostname):
-        ip_addrs.append(ipaddr_or_hostname)
+
+    # Convert the IP address to lowercase because IPv6 addresses will be stored
```

Comment:
Maybe change comment to `If we were passed an IP address, convert it to lowercase because...`